api/allennlp_demo/binary_gender_bias_mitigated_roberta_snli/requirements.txt

```
@@ -0,0 +1,4 @@
+git+git://github.com/allenai/allennlp.git@v2.5.0
+git+git://github.com/allenai/allennlp-models.git@v2.5.0
```

Might as well just use the PyPI release:

```suggestion
allennlp==2.5.0
allennlp-models==2.5.0
```

Better yet, if we update the default dependencies to point to AllenNLP v2.5.0 (https://github.com/allenai/allennlp-demo/blob/main/api/requirements.txt), then we can just use the default Dockerfile and can remove this requirements file as well as the Dockerfile next to it.

---

PR: https://github.com/allenai/allennlp-demo/pull/1021. Hopefully this doesn't break anything 😬